pytket/extensions/quest/quest_convert.py

```diff
@@ -14,10 +14,85 @@
 
 """Conversion from to tket circuits to QuEST circuits
 """
-from pytket.circuit import Circuit
+import numpy as np
+
+import pyquest.unitaries as gates
+from pyquest.gates import M as Measurement
+from pyquest import Circuit as PyQuESTCircuit
+
+from pytket.circuit import Circuit, OpType
+from pytket.passes import FlattenRegisters
+
+
+_ONE_QUBIT_GATES = {
+    OpType.X: gates.NOT,
+    OpType.Y: gates.Y,
+    OpType.Z: gates.Z,
+    OpType.H: gates.H,
+    OpType.S: gates.S,
+    OpType.T: gates.T,
+}
+
+_ONE_QUBIT_ROTATIONS = {OpType.Rx: gates.Rx, OpType.Ry: gates.Ry, OpType.Rz: gates.Rz}
+
+_MEASURE_GATES = {OpType.Measure: Measurement}
+
+_TWO_QUBIT_GATES = {OpType.CX: gates.X, OpType.CZ: gates.Z, OpType.SWAP: gates.Swap}
 
 
 def tk_to_quest(
     circuit: Circuit, reverse_index: bool = False, replace_implicit_swaps: bool = False
```

Comment:
I can see in the tests that with multi-qubit circuits we're basically always setting `reverse_index = True`. Does it make sense to make it `True` as default here?